app/services/idv/in_person_config.rb

```diff
@@ -1,7 +1,17 @@
 module Idv
   class InPersonConfig
     def self.enabled_for_issuer?(issuer)
-      enabled? && (issuer.nil? || enabled_issuers.include?(issuer))
+      return false if !enabled?
+
+      if issuer.nil?
+        enabled_without_issuer?
+      else
+        enabled_issuers.include?(issuer)
```

Comment:
separate convo (not a blocker for this PR), we're looking at moving these kinds of configs to the `service_providers` table right?

Reply:
Yep, that's tracked at LG-7203, which I expect will have us revisiting this code again.